ocsp/responder/multi_source.go

```diff
@@ -37,18 +38,26 @@ func NewMultiSource(primary, secondary Source, expectedFreshness time.Duration,
 	if primary == nil || secondary == nil {
 		return nil, errors.New("must provide both primary and secondary sources")
 	}
+
 	counter := prometheus.NewCounterVec(prometheus.CounterOpts{
 		Name: "ocsp_multiplex_responses",
 		Help: "Count of OCSP requests/responses by action taken by the multiSource",
 	}, []string{"result"})
 	stats.MustRegister(counter)
 
+	checkSecondaryCounter := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "ocsp_multiplex_check_secondary",
+		Help: "Count of OCSP requests/responses by action taken by the multiSource",
+	}, []string{"result"})
+	stats.MustRegister(counter)
```

Comment:
This should fix the integration test failure:
```suggestion
	stats.MustRegister(checkSecondaryCounter)
```